app/models/miq_worker/runner.rb

```diff
@@ -450,23 +450,22 @@ def clean_broker_connection
   def ruby_object_usage
     types = Hash.new { |h, k| h[k] = Hash.new(0) }
     ObjectSpace.each_object do |obj|
-      types[obj.class][:count] += 1
-      next if obj.kind_of?(DRbObject) || obj.kind_of?(WeakRef)
-      if obj.respond_to?(:length)
-        len = obj.length
-        if len.kind_of?(Numeric)
-          types[obj.class][:max]    = len if len > types[obj.class][:max]
-          types[obj.class][:total] += len
-        end
-      end
+      types[obj.class.name][:count] += 1
```

Comment:
You can also simplify this not to have the count at all, and thus not to have a sub-hash at all, since you aren't collecting any other statistics.